From: Kristian Nielsen <knielsen@knielsen-hq.org>
Date: Sun, 20 Jan 2019 20:59:05 +0100
Subject: Fix that csgtexttest does not find -lzip and -lz

This change is already upstream, so this patch can be removed after
2019.01-RC2.
--- a/tests/FindLib3MF.cmake
+++ b/tests/FindLib3MF.cmake
@@ -36,7 +36,7 @@
 endif()
 
 if (NOT ${LIB3MF_LIBDIR} STREQUAL "")
-  set(LIB3MF_LDFLAGS "-L${LIB3MF_LIBDIR}" "-l3MF")
+  set(LIB3MF_LDFLAGS "-L${LIB3MF_LIBDIR}" "-l3MF -lzip -lz")
   set(LIB3MF_CFLAGS "-D__GCC -DENABLE_LIB3MF")
   message(STATUS "Found lib3mf in ${LIB3MF_LIBDIR}.")
 else()
